lib/tracker/request-logger.js

```diff
@@ -0,0 +1,281 @@
+'use strict';
```

Comment:
Really dig the design of `RequestLogger` and `RequestTracker`, nice and clean, good documentation and easy to follow :+1: